website_sale_force_qty/models/sale_order.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    def _cart_update(self, product_id=None, line_id=None, add_qty=0, set_qty=0, **kwargs):
+        """Force quantity on product from what is defined in product.template"""
+        # Need to call super first to ensure correct default mechanism
+        res = super()._cart_update(product_id, line_id, add_qty, set_qty, **kwargs)
+        # Do nothing if we are removing the product (qty=0)
+        if res["quantity"] > 0 and product_id:
+            product = self.env["product.product"].browse(product_id)
```

Comment:
Should we add to the roadmap that qty forcing at variant level could be considered in the future?

Reply:
Yes, it could be a good idea. It would even help with the other point in the roadmap, as the variants have access to the quantity already in the cart, so we could use that to tweak the "Add to cart" button
@Yajo WDYT?